example/src/index.ts

```diff
@@ -161,9 +161,6 @@ timeGraphChart.registerMouseInteractions({
     click: el => {
         console.log('click: ' + el.constructor.name, el.model);
     },
-    doubleClick: el => {
-        console.log('doubleClick: ' + el.constructor.name, el.model);
-    },
```

Comment:
Please just amend these changes in the first commit. I don't think we need two separate commits?